### DISCUSSION TITLE
TypeError: searchInput.addValue is not a function

### DISCUSSION THREAD

**Discussion by mescaline007** [webdriverio/webdriverio#7062] [Q&A]
Started learning Webdrive, and stopped at this error:
![image](https://user-images.githubusercontent.com/19498794/123517437-96a2c980-d6a9-11eb-969b-ffc9e7ae7427.png)

Got it on any script. 

Dont know, how to resiolve it

**Comment by christian-bromann** [comment]
Can you provide an example? Make sure that `searchInput` is an actual WebdriverIO element and not an unresolved promise.

**Reply by mescaline007** [reply]
I've solve it. Problem was I've been using sync mode while it's not working

**Reply by mescaline007** [reply]
After editing sunction to async mode, everything is okay

**Reply by jaxaustin** [reply]
I am running into the same issue and figured it is due to sync mode. Is this mode held in the wdio.conf.js file somewhere? I can't find it but also did not set it during the 'npx wdio config' step as the option did not appear for me, I assume due to it no longer being used.

**Reply by AndruC** [reply]
> I am running into the same issue and figured it is due to sync mode. Is this mode held in the wdio.conf.js file somewhere? I can't find it but also did not set it during the 'npx wdio config' step as the option did not appear for me, I assume due to it no longer being used.

I believe if you have `@wdio/sync` installed in your `package.json`, you will be operating in synchronous mode. Remove it, and you'll be in asynchronous mode.
